.devcontainer/devcontainer.json

```diff
@@ -55,7 +55,8 @@
         }
     },
     "runArgs": [
-        "--cap-add=SYS_PTRACE"
+        "--cap-add=SYS_PTRACE",
+        "--privileged"
```

Comment:
Why do we need this?

Reply:
This was added to disable ASLR which was failing for me locally. I'll remove/git ignore it.

Reply:
Please remove this flag. One can disable ASLR globally 
```suggestion
```